### PR TITLE
feat/create utils package

### PR DESCRIPTION
**Pull request by PatrykKuniczak**
> `*` Please fill in the required items.

## Priority*

- [ ] High: This PR needs to be merged first for other tasks.
- [ ] Middle: This PR should be merged quickly to prevent conflicts due to common changes. (default)
- [x] Low: This PR does not affect other tasks, so it can be merged later.

## Purpose of the PR*
I want to create utils, like dev-utils which are already exists.

Users will have already predefined module for save them utils functions and types like this example "ValueOf"

## Changes*

Create 'utils' package and move "ValueOf" from 'dev-utils' to 'utils'

### CODE REVIEW

File: packages/utils/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@extension/utils",
+  "version": "0.2.2",
```
Comment (Jonghakseo) [inline packages/utils/package.json:3]:
```suggestion
  "version": "0.3.0",
```

File: packages/utils/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "index.ts",
+  "scripts": {
+    "clean": "rimraf ./dist && rimraf ./build",
+    "ready": "tsc",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "pnpm lint --fix",
+    "prettier": "prettier . --write --ignore-path ../../.prettierignore",
+    "type-check": "tsc --noEmit"
```
Comment (Jonghakseo) [inline packages/utils/package.json:19]: If we add functions in the future, we'll need to set esbuild or something... How about adding a function example?

**Review by Jonghakseo — COMMENTED** [review]
The definitions of packages inside monorepo seem to be getting a bit fuzzy and need to be clear.

There also seems to be some overlap with `shared`.

**Comment by PatrykKuniczak** [comment]
@Jonghakseo Maybe put this `utils` inside `shared` next to `hooks` ?

**Review by Jonghakseo — APPROVED** [review]
LGTM

**Pull request merged**